default-config.toml

```diff
@@ -55,6 +55,17 @@ emoji_favicons = false
 shift_numbers = "!@#$%^&*()"
 
 
+[url-handlers]
+# Allows setting the commands to run for various URL schemes.
+# E.g. to open FTP URLs with FileZilla set the following key:
+#   ftp = "filezilla"
+# You can set any scheme to "off" to disable handling it.
+
```

Comment:
```suggestion

# DO NOT use this for setting the HTTP command.
# Use the http setting in the "a-general" section above.
```

Remember to add this change to `default.go` as well.

Reply:
Good point!